tests/table/test_from_py.py

```diff
@@ -24,6 +24,7 @@
     "str": ["foo", "bar"],
     "binary": [b"foo", b"bar"],
     "date": [datetime.date.today(), datetime.date.today()],
+    "time": [datetime.time(1, 2, 3), datetime.time(4, 5, 6)],
```

Comment:
What happens if we try to construct a Daft table with a `datetime.time()` containing a timezone? I'm thinking that we should probably fail since a transparent conversion to e.g. UTC might cause the time to wrap around into a different logical day, which might be surprising.

Reply:
The timezone is ignored, looks like thats how pyarrow does it as well
```
In [5]: time_array = [datetime.time(1, 2, 3, 4, tzinfo=pytz.timezone("US/Eastern"))]

In [6]: daft.from_pydict({"time": time_array})
Out[6]: 
╭────────────────────╮
│ time               │
│ ---                │
│ Time(Microseconds) │
╞════════════════════╡
│ 01:02:03.000004    │
╰────────────────────╯

(Showing first 1 of 1 rows)

In [7]: pa.array(time_array, type=pa.time64("us"))
Out[7]: 
<pyarrow.lib.Time64Array object at 0x11fac3640>
[
  01:02:03.000004
]

```